bitnami/redmine/5/debian-11/rootfs/opt/bitnami/scripts/libredmine.sh

```diff
@@ -172,7 +172,7 @@ redmine_initialize() {
             redmine_conf_set "default.email_delivery.smtp_settings.user_name" "$REDMINE_SMTP_USER"
             redmine_conf_set "default.email_delivery.smtp_settings.password" "$REDMINE_SMTP_PASSWORD"
             # Remove 'USER@' part from e-mail address and use as domain
-            redmine_conf_set "default.email_delivery.smtp_settings.domain" "${REDMINE_SMTP_USER//*@/}"
+            redmine_conf_set "default.email_delivery.smtp_settings.domain" "${REDMINE_SMTP_DOMAIN:-${REDMINE_SMTP_USER//*@/}}"
```

Comment:
We should update this comment to reflect the possibility of using `$REDMINE_SMTP_DOMAIN`